stacks_queues/stack.py

```diff
@@ -21,18 +21,41 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store.head == None:
```

Comment:
Small style note: generally with what we call singletons (singleton classes are classes that can only have one instance of the class) like `None`, we prefer `is` to `==` because `==` can be overwritten with `__eq__` (just like you did with `__str__` in Queue, and as a result will very occasionally will lead to unexpected behavior particularly with singletons.
```suggestion
        if self.store.head is None:
```

Reply:
Wow, thank you! I was not aware of this!